packages/now-routing-utils/test/superstatic.spec.js

```diff
@@ -0,0 +1,267 @@
+const { deepEqual } = require('assert');
+const { normalizeRoutes } = require('../');
+const {
+  convertCleanUrls,
+  convertRedirects,
+  convertRewrites,
+  convertHeaders,
+  convertTrailingSlash,
+} = require('../dist/superstatic');
+
+function routesToRegExps(routeArray) {
+  const { routes, error } = normalizeRoutes(routeArray);
+  if (error) {
+    throw error;
+  }
+  return routes.map(r => new RegExp(r.src));
+}
+
+function assertMatches(actual, matches, isExpectingMatch) {
+  routesToRegExps(actual).forEach((r, i) => {
+    matches[i].forEach(text => {
+      deepEqual(r.test(text), isExpectingMatch, `${text} ${r.source}`);
+    });
+  });
+}
+
+function assertRegexMatches(actual, mustMatch, mustNotMatch) {
+  assertMatches(actual, mustMatch, true);
+  assertMatches(actual, mustNotMatch, false);
+}
+
+test('convertCleanUrls', () => {
+  const actual = convertCleanUrls([
+    'file.txt',
+    'path/to/file.txt',
+    'file.js',
+    'path/to/file.js',
+    'file.html',
+    'path/to/file.html',
+  ]);
+  const expected = [
+    {
+      src: 'file.html',
+      headers: { Location: 'file' },
+      status: 307,
+      continue: true,
```

Comment:
Redirects should not continue because `Location` will not merge

Reply:
Thanks. I removed all the `continue` props from anything returning headers.

Reply:
It makes sense with headers, just not `Location`